rebasehelper/versioneers/haskell_versioneer.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you to rebase package to the latest version
+# Copyright (C) 2013-2014 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#          Tomas Hozza <thozza@redhat.com>
+
+from rebasehelper.versioneer import BaseVersioneer
+from rebasehelper.logger import logger
+from rebasehelper.utils import DownloadHelper
+
+
+class HaskellVersioneer(BaseVersioneer):
+    NAME = 'haskell'
+    CATEGORIES = ['haskell']
+
+    BASE_URL = 'https://haskell.org'
+    API_URL = 'https://hackage.haskell.org'
+
+    @classmethod
+    def get_name(cls):
+        return cls.NAME
+
+    @classmethod
+    def get_categories(cls):
+        return cls.CATEGORIES
+
+    @classmethod
+    def _get_version(cls, package_name):
+        if package_name.startswith('ghc-'):
+            package_name = package_name.replace('ghc-', '', 1)
+
+        # Accept header is used to get JSON from Hackage API
```

Comment:
I think it's pretty self-explanatory, but if you think it's comment-worthy, then keep it.

Reply:
You are right, I will remove it.